apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts

```diff
@@ -267,6 +267,7 @@ export class ImportActivitiesDialog implements OnDestroy {
 
           return;
         } else if (file.name.endsWith('.csv')) {
+          const content: string[] = fileContent.split('\n').slice(1);
```

Comment:
Can you add a line break here?

Reply:
sure thing

Reply:
Hi @dtslvr by line break do u mean before line 270.